api/openapi.yaml

```diff
@@ -515,6 +515,53 @@ paths:
                 $ref: "#/components/schemas/StaticRouteItem"
         404:
           description: "Static Route not found by namespace-name combination"
+    put:
+      tags:
+        - "static routes"
+      summary: "Update an existing Static route"
+      description: "Updates an existing Static route in the cluster"
+      operationId: updateApi
+      parameters:
+        - name: namespace
+          in: path
+          required: true
+          schema:
+            type: string
+            default: "default"
+        - name: name
+          in: path
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: "Static route content that needs to be updated"
+        required: true
+        content:
+          application/json:
```

Comment:
@kylehodgetts since you are working on PUT for thi endpoint. Can you also review this and tweak it accordingly?